doc/source/data/inspecting-data.rst

```diff
@@ -143,3 +144,53 @@ of the returned batch, set ``batch_format``.
 For more information on working with batches, see
 :ref:`Transforming batches <transforming_batches>` and
 :ref:`Iterating over batches <iterating-over-batches>`.
+
+
+Inspecting runtime statistics
+=============================
+
+Ray Data calculates statistics during execution like runtime and memory usage for the different stages. 
+
+View debug stats for your Dataset executions via :meth:`Dataset.stats() <ray.data.Dataset.stats>` on an executed dataset. The stats are also persisted under `/tmp/ray/session_*/logs/ray-data.log`.
```

Comment:
What does "Dataset executions" mean? Does it mean transformations and reads and writes?

Reply:
updated